README.md

```diff
@@ -10,6 +10,8 @@ Currently, only Web pushes are supported.
 
 - Copy `.env.example` to `.env`
 - generate VAPID keys via `npx web-push generate-vapid-keys` and set Public Key to `PUBLIC_VAPID_KEY` and Private Key to `PRIVATE_VAPID_KEY` in `.env` file. The Public Key value should be the same to what is used in **sama-client**
+- add your firebase app name and apple keys to other variables
```

Comment:
set your Firebase app name in `FCM_APP_NAME` env and Apple keys in `APN_KEY_ID`, `APN_TEAM_ID` envs. Also provide `APN_IS_PRODUCTION` env